sections/main-cart-items.liquid

```diff
@@ -230,7 +236,15 @@
                               </button>
                             {%- endif -%}
                             <quantity-input class="quantity cart-quantity">
-                              <button class="quantity__button" name="minus" type="button">
+                              <button
+                                class="quantity__button"
+                                name="minus"
+                                type="button"
+                                {% assign can_update_quantity = item.instructions.can_update_quantity | default: true %}
+                                {% if can_update_quantity == false %}
+                                  disabled
+                                {% endif %}
+                              >
```

Comment:
(nit) I'd move the assign outside of the button, like this:
```suggestion
                              {% assign can_update_quantity = item.instructions.can_update_quantity | default: true %}
                              <button
                                class="quantity__button"
                                name="minus"
                                type="button"
                                {% if can_update_quantity == false %}
                                  disabled
                                {% endif %}
                              >
```

This way we don't risk it being undefined in the other uses if for some reason we remove or move the button to a different place.